test/testMethods.ts

```diff
@@ -253,23 +253,29 @@ module TestMethods {
     actualAreaPathStrings.pop();
     expectedAreaPathStrings.pop();
 
-    let actualAreaPathPoints = actualAreaPathStrings.map((path) => path.split(/[A-Z]/).map((point) => point.split(",")));
-    actualAreaPathPoints.forEach((areaPathPoint) => areaPathPoint.shift());
-    let expectedAreaPathPoints = expectedAreaPathStrings.map((path) => path.split(/[A-Z]/).map((point) => point.split(",")));
-    expectedAreaPathPoints.forEach((areaPathPoint) => areaPathPoint.shift());
-
-    assert.lengthOf(actualAreaPathPoints, expectedAreaPathPoints.length, "number of broken area paths should be equal");
-    actualAreaPathPoints.forEach((actualAreaPoints, i) => {
-      let expectedAreaPoints = expectedAreaPathPoints[i];
-      assert.lengthOf(actualAreaPoints, expectedAreaPoints.length, "number of points in path should be equal");
-      actualAreaPoints.forEach((actualAreaPoint, j) => {
-        let expectedAreaPoint = expectedAreaPoints[j];
-        assert.closeTo(+actualAreaPoint[0], +expectedAreaPoint[0], 0.1, msg);
-        assert.closeTo(+actualAreaPoint[1], +expectedAreaPoint[1], 0.1, msg);
-      });
+    let actualAreaPathNumbers = tokenizePathString(actualAreaPathStrings);
+    let expectedAreaPathNumbers = tokenizePathString(expectedAreaPathStrings);
+
+    assert.lengthOf(actualAreaPathNumbers, expectedAreaPathNumbers.length, "number of numbers in each path should be equal");
+    actualAreaPathNumbers.forEach((actualAreaNumber, i) => {
+      let expectedAreaNumber = expectedAreaPathNumbers[i];
+        assert.closeTo(+actualAreaNumber, +expectedAreaNumber, 0.1, msg);
     });
   }
 
+  function tokenizePathString(pathStrings: string[]) {
```

Comment:
Should be "`tokenizePathStrings`", based on the input, or it should take only one `string`. Is there a reason it takes an array?